README.md

```diff
@@ -8,14 +8,14 @@ Organization (mirroring) credentials are required to mirror SUSE repositories.
 
 ### SUSE products
 
-* Run `rmt-scc sync` to download available products and repositories data for your organization from SCC
-* Run `rmt-products list` to see the list of products that are available for your organization
-* Use the `rmt-repos` command to choose which product repositories to mirror, for example:
+* Run `rmt-cli scc sync` to download available products and repositories data for your organization from SCC
```

Comment:
Are there other subcommands expected for scc, or is it just ready for future expansion?

Reply:
Is 'nested RMT' a usecase?, i.e. is there a possibility that 'scc' in this case is not scc but another upstream RMT/SMT?

Reply:
You could check the RFC to see what is expected and what isn't :grimacing: 

No other subcommands planned at the moment. Chaining RMT servers also isn't on the backlog. However, disconnected setup (dumping SCC data into a file and importing it on a different machine) is a usecase -- coincidentally this is also used by SUMA.

I guess we can implement `export` and `import` subcommands rather than `-i` and `-o` flags for `rmt-cli scc sync`, as currently outlined by the RFC.